### PR TITLE
feat: create new filter for a10networks

### PR DESCRIPTION
**Pull request by ikheifets-splunk**

### CODE REVIEW

File: docs/sources/vendor/a10networks/vthunder.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+| source         | notes                                                                                                   |
+|----------------|---------------------------------------------------------------------------------------------------------|
+| a10networks:vThunder | None                                                                                              |
+
+### Index Configuration
+
+| key            | source         | index          | notes          |
+|----------------|----------------|----------------|----------------|
+| -| a10networks:vThunder| netwaf         | none           |
```
Comment (mstopa-splunk) [inline docs/sources/vendor/a10networks/vthunder.md:30]: key should be `vendor_product`, like: https://splunk.github.io/splunk-connect-for-syslog/main/sources/vendor/Citrix/netscaler/

File: docs/sources/vendor/a10networks/vthunder.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+## Key facts
+
+* MSG Format based filter
+* Legacy BSD Format default port 514
+
+## Links
+
+| Ref            | Link                                                                                                    |
+|----------------|---------------------------------------------------------------------------------------------------------|
+| Splunk Add-on  | None                                                                                                    |
```
Comment (mstopa-splunk) [inline docs/sources/vendor/a10networks/vthunder.md:12]: Not sure if community-owned add-ons count but there are some addons made by A10 networks that can be checked against regarding the sourcetype: https://splunkbase.splunk.com/apps?keyword=thunder

**Review by mstopa-splunk — CHANGES_REQUESTED** [review]

File: docs/sources/vendor/a10networks/vthunder.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+| source               | notes                                                                                             |
+|----------------------|---------------------------------------------------------------------------------------------------|
+| a10networks:vThunder | None                                                                                              |
+
+### Index Configuration
+
+| key        | source              | index          | notes          |
+|------------|---------------------|----------------|----------------|
+|a10_vthunder| a10networks:vThunder| netwaf         | none           |
```
Comment (mstopa-splunk) [inline docs/sources/vendor/a10networks/vthunder.md:33]: I failed to redirect event to another index with `splunk_metadata.csv` using this key:
```
a10_vthunder,index,main
```

Worked when changed to:
```
a10networks_vThunder,index,main
```

**Review by mstopa-splunk — APPROVED** [review]

**Review by mateuszpierzchala-splunk — APPROVED** [review]

**Pull request merged**